scripts/ansible/pool_get_logs.yml

```diff
@@ -30,3 +33,27 @@
       src: /var/log/indy/sandbox/
       dest: ./logs
```

Comment:
if it possible, please change to absolute path instead of relative

Reply:
Each scripts user forks indy-node in different place so probably it's better to keep relative paths for local machine.